integration-tests/manual-tests.md

```diff
@@ -198,3 +198,59 @@ $ snap remove bluez
    and verify it actually passes. If some of the tests fail
    there will be a problem with the particular kernel used on
    the device.
+
+# Test systemd-control interface
```

Comment:
Would be really cool to make this into a spread test, if you need help with that, please ping me on irc (its a bit tricky to test currently but we will have a qemu backend soon that makes testing a lot easier).